coredns-image/coredns-image.kiwi.ini

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _NAMESPACE_/coredns:%%LONG_VERSION%% _NAMESPACE_/coredns:%%LONG_VERSION%%-<RELEASE> -->
+
+<image schemaversion="6.5" name="_PRODUCT_-coredns">
```

Comment:
@drpaneas could you update the name image according the new criteria. Note that the `name` attribute needs to be the same name given to the build service project. This is a policy required for Factory and all other images have been adapted to that.